msb/config/MSBConfig.py

```diff
@@ -26,6 +26,9 @@ def __getitem__(self, key):
         else:
             warnings.warn(UserWarning(f"no such class member: {key}"))
 
+    def __repr__(self):
+        return json.dumps({key : self[key] for key in vars(self)}, indent=4)
+
```

Comment:
Why overwrite the default dataclass `__repr__`?
`__repr__` is for the class representation. If you want a json representation define a `to_json` method

Reply:
excellent idea - i've added it to the MSBConf base class